includes/breadcrumb.inc

```diff
@@ -98,10 +99,15 @@ EOQ;
     $filter_map = function ($filter) {
       return "FILTER($filter)";
     };
-    $query_filters[] = "sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOfCollection>) || sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOf>)";
+    $compound_object_predicate = variable_get('islandora_compound_object_relationship', 'isConstituentOf');
```

Comment:
Hi Jordan, thanks for your input, do you have any hook suggestion? I'm not aware of any existing hook that could permite us to get this predicate.